src/classLibrary/EvaluationStage/CyanParser.ts

```diff
@@ -0,0 +1,35 @@
+import { CyanObject, CyanSafe, ICyanParser, Syntax } from "../interfaces/interfaces";
+import _ from "lodash";
+
+class CyanParser implements ICyanParser {
+
+    Parse(cyanObject: Partial<CyanObject>): CyanSafe {
+        let defaultCyanSafe: CyanSafe = {
+            globs: [],
+            copyOnly: [],
+            variable: {},
+            flags: {},
+            guid: [],
+            syntax: [],
+            plugins: {},
+            comments: [],
+            pluginData: {}
+        }
+
+        if (cyanObject.globs) defaultCyanSafe.globs = _.concat([], cyanObject.globs);
```

Comment:
why concat?

Reply:
Concat used to force globs to become an array of globs, even if globs is an array or not an array